### PR TITLE
Removed dead link; Added new PDF

### PR DESCRIPTION
**Pull request by RallyTuning**
## What does this PR do?
Add resource(s) + Remove resource(s) + Add info

### Description
This is Visual Basic .Net, and not the old Visual Basic 6 (VB6). Also, this refer to the 2010 version (VB10) latest is the VB16 (2019). There wasn't so many changes since 2010, but some are relevant, like async and await. All changes from 2010 to 2019 can be see at this link: https://learn.microsoft.com/en-us/dotnet/visual-basic/whats-new
### Why is this valuable (or not)?
Is the only PDF about VB .Net and in italian language.
### How do we know it's really free?
It's old, no more in store.
### For book lists, is it a book? For course lists, is it a course? etc.
It's a book.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
(*) Not alphabetical because i just replaced the old line
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-it.md [hunk 1]
```diff
@@ -275,6 +275,6 @@
 * [Introduzione alla gestione del progetto software con UML](http://areaprofessional.com/documenti/D03_IntroGestioneProgettoSW.pdf) - G. Destri (PDF)
 
 
-### Visual Basic
+### Visual Basic .NET (VB .Net)
```
Comment (eshellman) [inline books/free-programming-books-it.md:278]: I'd leave the heading - if you change it you'll need to also change the index. also, we prefer broader categories

**Review by RallyTuning — COMMENTED** [review]
Index fixed

File: books/free-programming-books-it.md (hunk 1 above)
Comment (eshellman) [reply books/free-programming-books-it.md]: We like to keep headings consistent across our lists. we use just 'visual basic" in other lists. Put yourself in the shoes of a searcher. -will they type the extra  characters to fine only 'visual basic .net'?

Comment (RallyTuning) [reply books/free-programming-books-it.md]: Oh yes sorry, didn't know was also cross-list. Done

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**